go/ql/test/library-tests/semmle/go/aliases/InterfaceImpls/test.go

```diff
@@ -1,49 +1,56 @@
 package intfs
 
+// Tests that dataflow and interface implementation behave as expected when an interface
+// is implemented using a non-equal but identical type, i.e. an equal type after alias resolution.
```

Comment:
"Non-equal but identical" is a contradiction. Can we be more specific about the kinds of equality? Or perhaps we can get around the issue entirely with e.g.:
```suggestion
// Tests that dataflow and interface implementation behave as expected when an interface
// is implemented using different type names, which resolve to the same underlying type.
```